### DISCUSSION TITLE
Welcome to phenaki-pytorch Discussions!

### DISCUSSION THREAD

**Discussion by lucidrains** [lucidrains/phenaki-pytorch#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by juvekaradheesh** [comment]
Hey, 

I am a Ph.D. student at Virginia Tech working on Multimodal ML, and video generation. I greatly appreciate your work @lucidrains! I am currently using your video-diffusion-pytorch code and modifying it for my project.
